Cargo.toml

```diff
@@ -645,28 +645,42 @@ move-vm-test-utils = { path = "third_party/move/move-vm/test-utils", features =
 ] }
 move-vm-types = { path = "third_party/move/move-vm/types" }
 
+### Cargo profiles
+# See here for reference https://doc.rust-lang.org/cargo/reference/profiles.html
+
+# Release build
+# Major difference here is LTO which give a significant performance gain
 [profile.release]
+# Include debuginfo
 debug = true
+# Panic on overflow
 overflow-checks = true
+# Perform thin lto, achieving most of the benefit with less compile time
+lto = "thin"
+# Do not split crates into multiple compiled units
+codegen-units = 1
 
-# The performance build is not currently recommended
-# for production deployments. It has not been widely tested.
+# Correctness build, used for forge / testing
+# This is ostenisbly the same as release minus lto
+[profile.correctness]
+inherits = "release"
+lto = false
```

Comment:
let's remove for now, and add if we see the need?

Reply:
I like the way u roll

Reply:
I'm going to keep this in case, but we dont need to use it, useful for debugging non lto failures